src/Column/SynapseColumn.php

```diff
@@ -32,7 +32,7 @@ public function __construct(string $columnName, Synapse $columnDefinition)
      *     column_default:?string
      * } $dbResponse
      */
-    public static function createFromDB(array $dbResponse): self
+    public static function createFromDB(array $dbResponse): ColumnInterface
```

Comment:
Asi by nad tyma createFromDB měla byt anotace, že to vrací jen konkrétní implementaci :thinking: `@return SynapseColumn`

Reply:
hmm, to mi hazelo chybu, ze to neni kompatibilni s interface. Ale ted nehaze. Jeste to proverim. Ale taky by se mi to vice libilo

Reply:
Protože ti to jelo na PHP 7.1 ještě https://www.php.net/manual/en/language.oop5.variance.php

Reply:
jasne, tak to jeste prepisu

Reply:
myslim, ze lepsi https://github.com/keboola/php-table-backend-utils/pull/44/commits/3c4c973db0ff403c4cdd710735022a96144f041f